numpy/lib/histograms.py

```diff
@@ -207,7 +207,7 @@ def _hist_bin_fd(x, range):
     than the standard deviation, so it is less accurate, especially for
     long tailed distributions.
 
-    If the IQR is 0, this function returns 1 for the number of bins.
+    If the IQR is 0, this function returns 0 for the number of bins.
```

Comment:
```suggestion
    If the IQR is 0, this function returns 0 for the bin width.
```
`_hist_bin_fd` returns bin widths, not number of bins - this should clear up the issue

Reply:
gentle ping, can we accept this suggested change and merge ?

Reply:
Thanks for the ping, lets just do that.